src/lib/calculate-polygon-center.js

```diff
@@ -0,0 +1,165 @@
+import { Client } from "pg";
+const client = new Client();
+client.connect();
+
+// client.query(`CREATE TABLE Polygons (
+// 	type Text,
+// 	properties_name Text,
+// 	properties_type Text,
+// 	properties_pop2001 Numeric,
+// 	properties_dwell2001 Numeric,
+// 	properties_areaSqkm Numeric,
+// 	properties_popSqkm Numeric,
+// 	properties_province Text,
+// 	properties_centerLat Numeric,
+// 	properties_centerLng Numeric,
+// 	properties_lowLat Numeric,
+// 	properties_lowLng Numeric,
+// 	properties_highLat Numeric,
+// 	properties_highLng Numeric,
+// 	properties_radius Numeric,
+// 	geometry JSON
+// );`);
+
+client.query("SELECT * FROM myTable", (err, res) => {
+  const formattedPolygons = res.rows.map(row => {
+    const {
+      centerLat,
+      centerLng,
+      lowLat,
+      lowLng,
+      highLat,
+      highLng,
+      radius
+    } = getPolygonLimits({
+      coordinates: JSON.parse(row.geometrycoordinates),
+      geometryType: row.geometrytype
+    });
+
+    return {
```

Comment:
There is a function to convert the sql row to geojson, I believe its called geoJSON under utils. It doesn't have all the properties but I'd argue we extend that function and use it here :)

Reply:
Okay sick I'll take a look